src/pytorch_lightning/core/module.py

```diff
@@ -1730,64 +1697,6 @@ def optimizer_zero_grad(self, epoch, batch_idx, optimizer, optimizer_idx):
         """
         optimizer.zero_grad()
 
-    def tbptt_split_batch(self, batch: Any, split_size: int) -> List[Any]:
```

Comment:
Should we move this as a utility or get rid of it completely ?

Reply:
I agree with Adrian and would add a small deprecation just for this on the 1.9 branch.

The graveyard is not removed yet. There's an issue blocking the PR that I haven't been able to resolve